.github/workflows/nextjs_bundle_analysis.yml

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+name: 'Next.js Bundle Analysis'
+
+on:
+  pull_request:
+  push:
+    branches:
+      - dev
+
+defaults:
+  run:
+    # change this if your nextjs app does not live at the root of the repo
+    working-directory: ./
```

Comment:
This is the root, right?

Reply:
Yes, and I think its correct since the generated artifact includes all our routes and the global chunk size: https://github.com/safe-global/safe-wallet-web/actions/runs/6853583646

Reply:
I mean, you don't need to specify it as per the comment above.

Reply:
Ah ok, I will try to remove it completely and see if it works.

Reply:
Still works as expected 👍